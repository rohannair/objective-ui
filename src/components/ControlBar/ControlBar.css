@import '../../styles/globals';
@import '../../styles/mixins';
@import '../../styles/layout';

.controlBar {
  @extend row;
  @extend controlBar;
}
