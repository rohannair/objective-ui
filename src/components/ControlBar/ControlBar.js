import React, { PropTypes } from 'react'
import styles from './ControlBar.css'

const ControlBar = ({ children }) => (
  <div className={styles.controlBar}>
    { children }
  </div>
)

export default ControlBar
