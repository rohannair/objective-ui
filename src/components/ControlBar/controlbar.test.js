import React from 'react'

import ControlBar from './index.js'

describe('Test suite for ControlBar component', () => {
  it('ControlBar should exist', () => {
    let wrapper = shallow(<ControlBar/>)
    expect(wrapper).toMatchSnapshot()
  })
})

