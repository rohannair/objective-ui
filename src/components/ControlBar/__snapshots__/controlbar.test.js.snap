exports[`Test suite for ControlBar component ControlBar should exist 1`] = `<div />`;
